.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 *.log
 .env
 /build
+./userRouteNotes.js
```

Comment:
How will the other team get this if it's in gitignore? May as well remove, and remove the file, and send the file to them using conventional means, instead of polluting the gitignore file.